themes/bootstrap3/theme.config.php

```diff
@@ -45,6 +45,7 @@
         ['file' => 'lightbox.js', 'priority' => 320],
         ['file' => 'truncate.js', 'priority' => 330],
         ['file' => 'trigger_print.js', 'priority' => 340],
+        ['file' => 'resultcount.js', 'priority' => 350],
```

Comment:
Now that showCounts is configurable, would it make sense to conditionally load this via searchTabs.phtml when the feature is enabled, rather than always loading the Javascript?

Reply:
Sounds reasonable; I'm going to change this on monday.